config/portals.yml

```diff
@@ -64,6 +64,19 @@ portals: &portals
     scripts:
       - edsc-portal.cwic.js
 
+  IDN:
+    org: CEOS IDN
+    title: International Directory Network (IDN) Search
+    params:
+    	keyword:
+    		- *
+    
+    logos:
+      - id: CEOS-logo
+        image: /images/portals/idn/logo2.png
+        link: https://idn.uat.earthdata.nasa.gov
```

Comment:
This address doesn't seem to work -- is it down? Also, is it intentionally UAT?

Reply:
No, it can be Prod (OPS). After ESDIS review, the portal will be added to the CEOS IDN (https://idn.ceos.org/) site in  a click link box as a beta for review by the CEOS community.

Reply:
Can you update the value?

Reply:
@mmorahan, can you you please update the value here to point to prod?

Reply:
Made requested updates to the EDSC IDN Portal.